pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -15,10 +15,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	rbacv1client "k8s.io/client-go/kubernetes/typed/rbac/v1"
 	"k8s.io/kubernetes/pkg/apis/rbac"
+	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 
 	authorizationutil "github.com/openshift/origin/pkg/authorization/util"
+	userclientinternal "github.com/openshift/origin/pkg/user/generated/internalclientset"
```

Comment:
same comment about external clients.

Reply:
Replaced with `github.com/openshift/client-go/user/clientset/versioned/typed/user/v1`. I still see quite a few references to the internal user client in other parts of the codebase 🙁